dev/MRTCore/TestApps/winui_desktop_packaged_app/winui_desktop_packaged_app/App.xaml.cs

```diff
@@ -79,6 +90,30 @@ private void OnSuspending(object sender, SuspendingEventArgs e)
             // Save application state and stop any background activity
         }
 
+        private void SetWindowSize(IntPtr hwnd, int width, int height)
+        {
+            var dpi = GetDpiForWindow(hwnd);
+            float scalingFactor = (float)dpi / 96;
+            width = (int)(width * scalingFactor);
+            height = (int)(height * scalingFactor);
+
+            SetWindowPos(hwnd, System.IntPtr.Zero, 0, 0, width, height, 2);
+        }
+
+        [DllImport("user32.dll", SetLastError = true)]
+        static extern bool SetWindowPos(IntPtr hWnd, IntPtr hWndInsertAfter, int X, int Y, int cx, int cy, int uFlags);
```

Comment:
>SetWindowPos [](start = 27, length = 12)

it's odd to use Win32 API in a WinUI app, and it's in user32!

Reply:
That is the currently blessed way for doing this since the size APIs are not yet hooked up. I took this straight from the WinUI samples:)

Reply:
But yes, this is not ideal long-term. We will update it once WinUI has proper support.